config/default/common/config/metadata/layers/modis/LandSurfaceTemperature.md

```diff
@@ -7,6 +7,15 @@ The MODIS Land Surface Temperature product is available from both Terra (MOD11)
 
 References: MYD11_L2 [doi:10.5067/MODIS/MYD11_L2.061](https://doi.org/10.5067/MODIS/MYD11_L2.061); MOD11_L2 [doi:10.5067/MODIS/MOD11_L2.061](https://doi.org/10.5067/MODIS/MOD11_L2.061); [NASA Earth Observations - Land Surface Temperature](https://neo.sci.gsfc.nasa.gov/view.php?datasetId=MOD11C1_M_LSTDA)
 
+### MODIS (Terra/Aqua/ASTER) Land Surface Temperature (Day | Night)
+Temporal coverage: 8 July 2021 - present (Terra); 9 July 2021 - present (Aqua)
+
+The Land Surface Temperature (Day | Night) layer is a new Land Surface Temperature product (MOD/MYD21), in addition  to the heritage MOD11/MYD11 Land Surface Temperature (LST) product. It show the temperature of the land surface in Kelvin (K). The MOD21 Temperature/Emissivity Separation (TES) algorithm uses a physics-based algorithm to dynamically retrieve both LST and spectral emissivity simultaneously from the MODIS thermal infrared bands 29, 31, and 32. The TES algorithm is combined with an improved Water Vapor Scaling (WVS) atmospheric correction scheme to stabilize the retrieval during very warm and humid conditions. The MOD/MYD21 Land Surface Temperature (LST) algorithm differs from the MOD11/MYD11 L2 algorithm in that the MOD21/MYD21 LST algorithm is based on the ASTER Temperature/Emissivity Separation (TES) technique, whereas the MOD11/MYD11 L2 products uses the split-window technique.
```

Comment:
There's an extra space here between addition and to : `in addition  to`

Reply:
fixed!